shared/src/prompts.rs

```diff
@@ -58,6 +58,43 @@ pub fn add_cidr(cidrs: &[Cidr], request: &AddCidrOpts) -> Result<Option<CidrCont
     )
 }
 
+/// Bring up a prompt to delete a CIDR. Returns the peer request.
+pub fn delete_cidr(cidrs: &[Cidr], peers: &[Peer], request: &DeleteCidrOpts) -> Result<i64, Error> {
+    let eligible_cidrs: Vec<_> = cidrs
+        .iter()
+        .filter(|cidr| {
+            cidr.name != "innernet-server" &&
```

Comment:
I think we can remove this check for "innernet-server" here since it will never be eligible for deletion.

```suggestion
```

Reply:
Great point!